modulegen/internal/dependabot/main.go

```diff
@@ -0,0 +1,69 @@
+package dependabot
+
+import (
+	"fmt"
+
+	"github.com/testcontainers/testcontainers-go/modulegen/internal/context"
+)
+
+// Generator is a struct that contains the logic to generate the dependabot config file.
+type Generator struct{}
+
+// AddModule update dependabot with the new module
+func (g Generator) AddModule(ctx context.Context, tcModule context.TestcontainersModule) error {
+	configFile := ctx.DependabotConfigFile()
+
+	config, err := readConfig(configFile)
+	if err != nil {
+		return fmt.Errorf("read config: %w", err)
+	}
+
+	if err := config.addUpdate("/" + tcModule.ParentDir() + "/" + tcModule.Lower()); err != nil {
```

Comment:
question: should this be filepath.Join?

Reply:
Indeed, I'm changing it now, although the first element must be the `filepath.Separator`(see dependabot.yaml, the directories for an update)